source/includes/common/common-minio-kes-aws.rst

```diff
@@ -33,6 +33,9 @@ Manager:
        - /v1/key/create/*
        - /v1/key/generate/*
        - /v1/key/decrypt/*
+       - /v1/key/list*
+       - /v1/status
+       - /v1/metrics
```

Comment:
```
      - /v1/key/create/*
      - /v1/key/generate/*
      - /v1/key/decrypt/*
      - /v1/key/encrypt/*
      - /v1/key/bulk/decrypt
      - /v1/key/list
      - /v1/status
      - /v1/metrics
      - /v1/log/audit
      - /v1/log/error
```

to provide all the `admin` permissions.

Reply:
At this point it's probably not worth trying to do a minimal set, so I'll add those in throughout. Thanks!

Reply:
@harshavardhana LMK if you have any other comments, otherwise I'll merge EOD